objects/description/include/object_description.h

```diff
@@ -10,10 +10,45 @@
 #ifndef OBJECT_DESCRIPTION_H_
     #define OBJECT_DESCRIPTION_H_
 
+    #include <stddef.h>
+
+    ///
+    /// \brief Error inside a Description object
+    ///
+    #define DESCRIPTION_ERROR NULL
+
+    ///
+    /// \brief Max description size
+    ///
+    #define DESCRIPTION_SIZE 255
+
+    ///
+    /// \brief Description error message for a bad size
+    ///
+    #define DESCRIPTION_INVALID_SIZE "Invalid Description size."
+
 ///
 /// \brief Description object equivalent of a string
 ///
 ///
-typedef char description_t[255];
+typedef char *description_t;
+
+///
+/// \brief Init the description object
+/// If the description_content is too long compared to description_size
+/// return DESCRIPTION_ERROR
```

Comment:
It's about return value, move it to \return

Reply:
Fix in the next PR